project_scrum/views/project_project.xml

```diff
@@ -58,6 +58,22 @@
                     <field name="meeting_count" widget="statinfo" />
                 </button>
             </button>
+            <xpath
+                expr="//field[@name='task_count']/parent::button"
```

Comment:
This Xpath is correct but you can identify it better. could had identified the button directly even if there is no ID on the object, by xpathing the action_id. less prone to errors and future breakage.

Reply:
Still can identify object by the button action property, without using parent.